analyzer.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+# for details. All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+runtime: custom
+env: flex
+service: analyzer
+env_variables:
```

Comment:
That is a well-hidden env variable. I was looking for similar, but was without luck, and I have found little reference even after I knew what to search for... fix in progress